bigquery_storage/setup.cfg

```diff
@@ -1,10 +1,3 @@
+# Generated by synthtool. DO NOT EDIT!
 [bdist_wheel]
 universal = 1
-
-[flake8]
-exclude =
-    *_pb2.py
-    *_pb2_grpc.py
-    google/cloud/bigquery_storage_v1beta1/gapic/big_query_storage_client.py
-    google/cloud/bigquery_storage_v1beta1/gapic/transports/big_query_storage_grpc_transport.py
-    tests/unit/gapic/v1beta1/test_big_query_storage_client_v1beta1.py
```

Comment:
Is this change intentional?  Seems like it might break stuff.

Reply:
Linter is passing. Seems these lines are no longer needed.